libr/anal/function.c

```diff
@@ -52,6 +52,8 @@ static bool __fcn_exists(RAnal *anal, const char *name, ut64 addr) {
 	return false;
 }
 
+void r_anal_var_free(RAnalVar *av);
```

Comment:
this should be in the .h as R_API

Reply:
No, it's private and must never be used outside the function.c or var.c, otherwise you are doing something wrong.

Reply:
then use R_IPI (internal private ipa)

Reply:
Yeah it's an international penile inspection.